io.github.YouROK.TorrServer.yml

```diff
@@ -0,0 +1,39 @@
+app-id: io.github.YouROK.TorrServer
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: run.sh
+finish-args:
+  - --share=network
```

Comment:
this displays nothing?

Reply:
yes. it's a terminal app

Reply:
terminal apps really are not the use case for Flatpak.

Reply:
@hfiguiere I guess you could contribute here then https://unix.stackexchange.com/questions/740712/does-flatpak-support-command-line-applications

Reply:
No.

Reply:
You can remove that arg and apply for an exception https://docs.flathub.org/docs/for-app-authors/linter#finish-args-not-defined